src/seedu/addressbook/Main.java

```diff
@@ -55,18 +58,22 @@ private void start(String[] launchArgs) {
             this.addressBook = storage.load();
             ui.showWelcomeMessage(VERSION, storage.getPath());
 
-        } catch (InvalidStorageFilePathException | StorageOperationException e) {
+        } catch (InvalidStorageFilePathException | StorageOperationException | FileNotFoundException fnfe) {
```

Comment:
the problem of file going missing is not limited to just the initialization stage.
hence, you need to handle the exception little better than combining it here :-)